### PR TITLE
add support for building RPMs

### PR DESCRIPTION
**Pull request by drossetti**
resolve Issue #7

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 PREFIX ?= /usr/local
+DESTLIB ?= /usr/local/lib64
```
Comment (drossetti) [inline Makefile]: this is not correct, if PREFIX is passed but DESTLIB is not, we want DESTLIB to change accordingly (appending /lib64 to PREFIX)

**Comment by drossetti** [comment]
@spotluri there is no reliable way of expressing that requirement. E.g. when nvidia driver is installed via the .run installer, there is no init.d script associated with it.

File: Makefile (hunk 1 above)
Comment (drossetti) [inline Makefile]: fixed in latest push

File: init.d/gdrcopy [hunk 2]
```diff
@@ -78,6 +91,12 @@ load_module()
 # Start daemon
 function start() {
 
+    echo -n $"Checking required modules: "
```
Comment (drossetti) [inline init.d/gdrcopy]: @spotluri with this change, the init script will check for the presence of the nvidia driver

**Comment by drossetti** [comment]
@spotluri fyi locally tested on both RH6 and 7

File: init.d/gdrcopy (hunk 2 above)
Comment (spotluri) [inline init.d/gdrcopy]: thanks @drossetti!

**Pull request merged**